### PR TITLE
change: MainConcept Transcoders update

### PR DESCRIPTION
**Pull request by jongov**
MainConcept Transcoders update

## Description 📝
MainConcept Transcoders links and name update

## Changes  🔄
List any change relevant to the reviewer.
- MainConcept Transcoders Link Updated
- MainConcept P2 AVC-ULTRA name updated

### CODE REVIEW

File: packages/manager/src/features/OneClickApps/oneClickApps.ts [hunk 1]
```diff
@@ -1173,20 +1173,20 @@ export const oneClickApps: OCA[] = [
       end: '041125',
       start: '6DBA98',
     },
-    description: `MainConcept P2 AVC Ultra Transcoder is an optimized Docker container for file-based transcoding of media files into professional Panasonic camera formats like P2 AVC-Intra, P2 AVC LongG and AVC-intra RP2027.v1 and AAC High Efficiency v2 formats into an MP4 container.`,
+    description: `MainConcept P2 AVC-ULTRA Transcoder is an optimized Docker container for file-based transcoding of media files into professional Panasonic camera formats like P2 AVC-Intra, P2 AVC LongG and AVC-intra RP2027.v1 and AAC High Efficiency v2 formats into an MP4 container.`,
     logo_url: 'mainconcept.svg',
-    name: 'MainConcept P2 AVC Ultra Transcoder',
+    name: 'MainConcept P2 AVC-ULTRA Transcoder',
```
Comment (cpathipa) [inline packages/manager/src/features/OneClickApps/oneClickApps.ts:1178]: Unable to find this in the search results; I think it's due to the hyphen.

image

Comment (jongov) [reply packages/manager/src/features/OneClickApps/oneClickApps.ts]: Hyphen removed.

**Comment by jongov** [comment]
Hyphen removed from MainConcept P2 AVC-ULTRA Transcoder name.

**Comment by jdamore-linode** [comment]
Hey @jongov, thank you for the contribution and sorry for my delay responding!

Would you mind providing a little bit of context for this change? For example, both the `/mc2go` and `/transcoders` webpages seem to contain similar content, and I see the product name stylized as _MainConcept P2 AVC Ultra_ and _MainConcept P2 AVC-ULTRA_ throughout the website, so I'm just curious about the motivation for this change.

Thanks again!

**Comment by jongov** [comment]
Hi @jdamore-linode ,

MC2GO products have been renamed to Transcoders. The /mc2go page will be removed soon and only /transcoders will remain. 

On the P2 AVC-ULTRA name - this is the official name of the Panasonic format and how Panasonic recommend to be written - all caps with hyphen. There was an issue in the Linode platform with that hyphen so I removed it, but better to stick to the Panasonic's recommendation where possible.

**Comment by jdamore-linode** [comment]
> Hi @jdamore-linode ,
> 
> MC2GO products have been renamed to Transcoders. The /mc2go page will be removed soon and only /transcoders will remain.
> 
> On the P2 AVC-ULTRA name - this is the official name of the Panasonic format and how Panasonic recommend to be written - all caps with hyphen. There was an issue in the Linode platform with that hyphen so I removed it, but better to stick to the Panasonic's recommendation where possible.

Understood. Thanks for the explanation, @jongov!

**Review by jdamore-linode — APPROVED** [review]

**Review by bnussman-akamai — APPROVED** [review]

**Pull request merged**